### PR TITLE
GRA-1384: removed join by access key

### PR DESCRIPTION
**Pull request by 0xdcarns**
test with corresponding server branch, join should yield same result as register (with -t)

### CODE REVIEW

**Review by afeiszli — APPROVED** [review]

File: functions/register.go [hunk 1]
```diff
@@ -68,3 +72,52 @@ func Register(token string) error {
 	fmt.Printf("registered with server %s\n", serverData.Server)
 	return nil
 }
+
+func doubleCheck(host *config.Config, apiServer string) (shouldUpdate bool, err error) {
```
Comment (mattkasun) [inline functions/register.go:76]: This function is not required.  CheckConfig is callled at start of every netclient command; if there is a check here that is not in that function, it should be added to that function

File: gui/exported.go [hunk 2]
```diff
@@ -119,40 +114,40 @@ func (app *App) GoGetRecentServerNames() ([]string, error) {
...
-	config.InitConfig(flags)
-	err := functions.Join(flags)
-	if err != nil {
-		fmt.Println(err)
-		return nil, err
-	}
-
-	return nil, nil
-}
+// // App.GoJoinNetworkBySso joins a network by SSO
```
Comment (mattkasun) [inline gui/exported.go:117]: delete rather than comment out

**Review by mattkasun — CHANGES_REQUESTED** [review]

File: functions/register.go (hunk 1 above)
Comment (0xdcarns) [reply functions/register.go:76]: this was added as duct tape a while ago, somehow some of these host fields were not populated. maybe you can fix checkconfig in future, leaving for now.

File: gui/exported.go (hunk 2 above)
Comment (0xdcarns) [reply gui/exported.go:117]: will do

**Review by abhishek9686 — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
pr steps pass
critical checklist 100%

**Pull request merged**